modules/tensor_mechanics/src/auxkernels/CylindricalRankTwoAux.C

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "CylindricalRankTwoAux.h"
+
+template<>
+InputParameters validParams<CylindricalRankTwoAux>()
+{
+  InputParameters params = validParams<AuxKernel>();
+  params.addClassDescription("Takes RankTwoTensor material and outputs component in cylindrical coordinates");
+  params.addRequiredParam<MaterialPropertyName>("rank_two_tensor", "The rank two material tensor name");
+  params.addRequiredRangeCheckedParam<unsigned int>("index_i", "index_i >= 0 & index_i <= 2", "The index i of ij for the tensor to output (0, 1, 2)");
+  params.addRequiredRangeCheckedParam<unsigned int>("index_j", "index_j >= 0 & index_j <= 2", "The index j of ij for the tensor to output (0, 1, 2)");
```

Comment:
This may sound nitpicky, but I'm actually curious, the ```>= 0``` can never be violated for an `unsigned int` right? The parameter is read in as unsigned from the get go, so a negative number would be represented as intmax - something.